app/models/user/reputation_token.rb

```diff
@@ -23,6 +23,9 @@ class User::ReputationToken < ApplicationRecord
     self.reason = self.class.instance_variable_get("@reason")
     self.category = self.class.instance_variable_get("@category")
 
+    # TODO: This should be set correctly in the command not here
+    self.earned_on = Time.current.to_date unless earned_on
```

Comment:
@ErikSchierboom We're going to want to set this to be the point at which a PR is merged (or equivalent for other things)

Reply:
Okay. Want me to add that to this PR?

Reply:
I think a second PR.